src/layout/Header.js

```diff
@@ -73,6 +73,7 @@ function Header({ noMargin, noBackground, positionAbsolute }) {
             <NavLink to="/events">Events</NavLink>
             <NavLink to="/pricing">Pricing</NavLink>
             <GuideDropdown />
+            <LogoutButton />
```

Comment:
<img width="180" alt="Screenshot 2019-12-06 at 15 20 01" src="https://user-images.githubusercontent.com/2630/70333830-210fa800-183c-11ea-8f11-6d86c00b18bb.png">

Needs a space between "Logout" and  Signinbutton